README.md

```diff
@@ -1 +1,160 @@
+
+### Key Features <a name="key-features"></a>
+
+- User-Friendly Interface
+- Authentication
+- Authorization
+- Creating Transaction Categories
+- Adding New Transactions
+- Displaying Transaction Details
+- Total Amount Calculation for each Category
+- Ability to Add/Edit/Delete Transactions
+- Link to Add a New Transaction for each Category
+- Navigation Back to Categories Page
+- Transaction Entries Displayed in a Card Layout
+
+<!-- LIVE DEMO -->
+
+### Live <a name="live"></a>
+
+<a href='https://transaction-app-m02x.onrender.com/'>Click Here</a>
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project, you need:
+  - A browser of your choice.
+  - A text editor of your choice.
+  - An installed Ruby gem on your local system.
+  - Rails needs to be installed on your system.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Use the following commands:
+  - git clone https://github.com/MMhaidari/rails-solo-capstone.git
+  - cd rails-group-capstone
+
+### Install
+  - cd rails-solo-capstone
+  - bundle install (install all dependencies)
+
+### Usage
+
+- bin/rails server (to run the web application)
+
+### Run tests
+- Run the following script and style test:
+  - rspec spec ./name_of_file_spec.rb
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Mohammad Mohsen Haidari**
+
+- GitHub: [Mohsen](https://github.com/MMhaidari)
+- LinkedIn: [Mohsen](https://www.linkedin.com/in/mohammad-mohsen-haidari/)
+- Twitter: [Mohsen](https://twitter.com/MMhaidari12)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Ability to upload and download videos of Recipes and Foods**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/MMhaidari/rails-solo-capstone/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, give it a star and share it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank Microverse and coding partners for the commitment and dedication to working and finishing the project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> I would like to thank [Microverse Inc](https://www.github.com/microverseinc) for providing the project requirements
+
+> I would like to thank [Behance](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) for providing the project design template
```

Comment:
- [ ] the acknowledgment sections is double in your readme file right now, please remove one of them.
- [ ] also make sure to give credit to Gregoire Vella on Behance the author of the original design, instead of Behance itself